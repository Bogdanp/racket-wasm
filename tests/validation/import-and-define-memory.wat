(module
 (import "a" "m" (memory 1))
 (memory 1))