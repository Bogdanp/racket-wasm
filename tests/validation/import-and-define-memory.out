(#f . "only one memory block allowed\n  at: (memtype (limits 1 #f))")